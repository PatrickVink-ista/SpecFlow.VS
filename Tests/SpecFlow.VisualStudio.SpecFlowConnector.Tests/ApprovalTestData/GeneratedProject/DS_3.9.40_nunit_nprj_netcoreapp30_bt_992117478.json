{
    "GeneratorOptions": {
        "featureFileCount": 1,
        "scenarioPerFeatureFileCount": 1,
        "newProjectFormat": true,
        "targetFramework": "netcoreapp30",
        "SpecFlowPackageVersion": "3.9.40"
    }
}
