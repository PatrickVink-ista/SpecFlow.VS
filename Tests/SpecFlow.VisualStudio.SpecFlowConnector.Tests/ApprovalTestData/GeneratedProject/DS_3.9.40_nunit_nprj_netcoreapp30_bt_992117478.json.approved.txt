{
  "stepDefinitions": [
    {
      "type": "Given",
      "regex": "^quam Donec porta Sed$",
      "method": "IntegerANecHimenaeosSteps.GivenCursusMolestieRisusUltricies(String)",
      "paramTypes": "s",
      "expression": "quam Donec porta Sed",
      "sourceLocation": "#0|10|9|12|10"
    },
    {
      "type": "Then",
      "regex": "^per ipsum (\\d\u002B)$",
      "method": "IntegerANecHimenaeosSteps.ThenAcTortorEgestasVel(Int32)",
      "paramTypes": "i",
      "expression": "per ipsum (\\d\u002B)",
      "sourceLocation": "#0|16|9|18|10"
    },
    {
      "type": "When",
      "regex": "^sapien (\\d\u002B) ante nulla$",
      "method": "IntegerANecHimenaeosSteps.WhenEnimRisusCongueEt(Int32)",
      "paramTypes": "i",
      "expression": "sapien (\\d\u002B) ante nulla",
      "sourceLocation": "#0|22|9|24|10"
    }
  ],
  "sourceFiles": {
    "0": "<<targetFolder>>\\StepDefinitions\\IntegerANecHimenaeosSteps.cs"
  },
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "TechTalk.SpecFlow.dll",
    "SFFileVersion": "3.9.40.46609",
    "SFProductVersion": "3.9.40\u002B11b6bbec89",
    "SourcePaths": "1",
    "StepDefinitions": "3",
    "TargetFramework": ".NETCoreApp,Version=v3.0",
    "TypeNames": "0"
  }
}
